libs/logging/src/log_event_ids.ts

```diff
@@ -39,23 +37,15 @@ export enum LogEventId {
   UsbDriveMounted = 'usb-drive-mount-complete',
```

Comment:
There are a handful of events like this where the enum member (`UsbDriveMounted`) differs from the value (`'usb-drive-mount-complete'`) in more than just formatting (the addition of the `-complete` suffix). Ideally if formatting was the only difference, we could do away with the `eventId` field in the `.toml` file and just convert between title case and dash/kebab-case.

Since we'd have to update callsites of those logs it's probably safer to do it in another PR.